test/test_regression_cli.py

```diff
@@ -59,9 +59,12 @@ def setup_module():
 
 def teardown_module():
     """Cleanup temporary output files after testing and return to root directory"""
+    os.system("rm -r output*")
     os.system("rm output*")
```

Comment:
I couldn't remove `output_json.csv` and `output_xml.csv` from these commands. Any idea how to remove those as well?